.evergreen/config.yml

```diff
@@ -481,8 +481,8 @@ functions:
           export AWS_DEFAULT_REGION="us-east-1"
 
           # Set client-side encryption credentials.
-          export CSFLE_TLS_CA_FILE="$DRIVERS_TOOLS/.evergreen/x509gen/ca.pem"
-          export CSFLE_TLS_CERTIFICATE_KEY_FILE="$DRIVERS_TOOLS/.evergreen/x509gen/client.pem"
+          export CSFLE_TLS_CA_FILE="$PROJECT_DIRECTORY/data/kmip-certs/ca-ec.pem"
```

Comment:
What does `$PROJECT_DIRECTORY` reference? Does this correspond with an update to the evergreen host builds?